### ISSUE TITLE
Accessibility issue when a product have more than 1 image.

### ISSUE THREAD

**Issue by betayazilim** [woothemes/woocommerce#44478]
### Prerequisites

- [X] I have carried out troubleshooting steps and I believe I have found a bug.
- [X] I have searched for similar bugs in both open and closed issues and cannot find a duplicate.

### Describe the bug

When there is more than one image, product page have accessibility issue.
Values assigned to role="" are not valid ARIA roles.
img.zoomImg css class probably

### Expected behavior

There should be no error

### Actual behavior

https://pagespeed.web.dev/analysis/https-themes-woo-com-storefront-product-build-your-dslr/3zcvlwuhd6?form_factor=mobile

Values assigned to role="" are not valid ARIA roles. on Accessibility section

### Steps to reproduce

1. Go to product page which has more than 1 image
2. Copy url and paste pagespeed tool or right click and use lighthouse tool
Here is an example

### WordPress Environment

All latest.
But even storefront theme have it on woo.com

### Isolating the problem

- [X] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [X] This bug happens with a default WordPress theme active, or [Storefront](https://woo.com/products/storefront/).
- [X] I can reproduce this bug consistently using the steps above.

**Comment by Aljullu** [comment]
Thanks for opening this issue, @ned-bs! I was able to reproduce with following these steps:

1. Create a new store and import the sample products.
2. Go to Media > Library and add an `alt` text to all _Hoodie_ images.
3. Use Google PageSpeed Insights to analyze `www.example.com/product/hoodie/`.
4. One of the errors returned mentions: `Values assigned to role="" are not valid ARIA roles`

I'm a bit confused because [`role="presentation"`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/presentation_role) should be a valid role. It's used in the zoomed-in image and I think it makes sense to hide it from assistive technologies, given that the non-zoomed-in image is already provided to them. One easy thing we could try is replacing `role="presentation"` with `aria-hidden="true"`, which in this context has the same meaning and might work better for the PageSpeed Insights.

@nathanss I see you this issue is assigned to Mothra, but given that it makes reference to the classic frontend, I wonder if it would be better for Kirigami & Origami, what do you think?

**Comment by amesplant** [comment]
@Manussakis since you are working on this one, you might want to address this https://github.com/woocommerce/woocommerce/issues/48388 as well 😁
